_ru/overviews/collections/introduction.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: multipage-overview
+title: Введение
+
+discourse: false
+
+partof: collections
+overview-name: Collections
+
+num: 1
+
+language: ru
+---
+
+**Martin Odersky и Lex Spoon**
+
+Популярно мнение, что новые коллекции это самое значимое изменение в Scala версии 2.8. 
+В Scala и раньше были коллекции (и на самом деле, новая модель
+во многом совместима с прежней). Но только начиная с версии 2.8 
+появилась единая, общая и всеобъемлющая база для всех типов коллекций.
+
+Несмотря на то, что улучшения коллекций на первый взгляд едва заметны, они могут привести к значительным изменениям в вашем стиле программирования.  
+Теперь разработка программы выглядит больше, как операции на более высоком уровне, где вся программа будто одна целая коллекция, в отличии от программирования на уровне комбинирования отдельных элементов коллекций. 
```

Comment:
In fact, quite often it's as if you work on a higher-level with the basic building blocks of a program being whole collections instead of their elements

`basic building blocks of a program` != `program`. `основными блоками для построения программ`?